app/src/main/res/values/strings.xml

```diff
@@ -677,6 +686,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="contributions_of_user">Contributions of User: %s</string>
   <string name="achievements_of_user">Achievements of User: %s</string>
   <string name="menu_view_user_page">View user page</string>
+  <string name="edit_depicts">Edit Depicts</string>
```

Comment:
Edit Depicts -> Edit depictions

It is the same as depict_edit_button_text so maybe factorize and use only one string? This will save our translators a lot of effort.